remo/profiles/fixtures/demo_users.json

```diff
@@ -167,7 +167,7 @@
         "fields": {
             "city": "Athens",
             "twitter_account": "",
-            "display_name": "admin",
+            "display_name": "admin-1",
```

Comment:
This seems to be the exact same entry as the first admin one. It even has the same "primary key". Can you delete the full entry here and see if it still works? I don't think we will need it in the future.

Reply:
Actually, nevermind. It's not fully the same and this file is a mess anyway.

Reply:
Can you please provide a short summary on how the fixtures are not loading? I cannot replicate this locally. Moreover, the `display_name` field is different than the `username` which uses the `admin` keyword and there is no other entry with `admin` as the display name. The only explanation that I can think of that could cause a collision is if there is already another entry with this display_name in the db prior to the fixture loading.

Reply:
@CuriousLearner check this one so we will have soon the merge :-)